### ISSUE TITLE
Anyone got a working HelloWorld.html POC to Copy and Paste? 

### ISSUE THREAD

**Issue by MichaelCropper** [bramp/js-sequence-diagrams#240]
Feels like I've tried every combination and permutation documented and can't even get a basic version working. I'm getting the feeling that this is now a dead project, which is a real shame if that's the case as it's a really handy tool to use via the demo platform, but not much use if can't have a play locally.

**Comment by bramp** [comment]
Have you tried copy and pasting the demo website? And using that as your hello world?

But admittedly not updated this project in quite some time, but it's love to see what errors you get with the simplest setup.

**Comment by MichaelCropper** [comment]
Yeah I did the old fashioned Right Click + Download Website. Funny enough, that is the closest working example I got. But it didn't work. 

I've tried going down various debugging routes to see if it was something simple to fix but nothing I tried to fix the errors in the Chrome Console / Network Tab worked. 

I really don't want to write off this project as it has such potential (I've used the web demo version for years)

Perhaps it will stand out to you @bramp what the issue could be having built it? 

If I can get a basic POC working, that's all I need to have a play further.

**Comment by bramp** [comment]
If it helps the site's source is on GitHub, so you don't have to right click save :-) https://github.com/bramp/js-sequence-diagrams/blob/gh-pages/index.html 

If I get a chance I will go though the readme and make sure the instructions work in the modern era

**Comment by MichaelCropper** [comment]
Yeah, I went through that too - the Git Clone process, the Node + NPM + Bower + Make processes as described and absolutely everything errored in one way or another. Feels like it's probably a fairly simply fix, if you know what you're looking for - but I just don't have the patience to try and reverse engineer an entire project that is over 10 years old. 

Give me a nudge once the Readme has had a spring clean and I'll give it another go. 

All I'm looking for is something I can copy and paste into Notepad++;
 - HelloWorld.html
 - Basic JavaScript <script> tags and CSS <link> tabs for Includes from CDN / GitHub RAW URLs

If people can get things working at this basic level, they can worry about all the magic package managers and command line stuff much later down the line and based on their tech preferences / tool stack etc. 

i.e. For a basic example in the documentation with no;
 - NPM Dependency
 - Bower Dependency
 - Make Dependency
 - Git Dependency

Just a real back to basics POC example

**Comment by MichaelCropper** [comment]
I've just checked that link and noticed it's not the Master branch. I've just checked out the "gh-pages" branch and that's working now with a basic Git Clone / Download as Zip. 

Excellent, that should keep me busy for a while to have a play around now. 

Might be worth just updating the Readme on the Master Branch that this is how to quickly create a POC so people can have a play.

**Comment by MichaelCropper** [comment]
I've just submitted a PR as a quick update for future readers - https://github.com/bramp/js-sequence-diagrams/pull/241

**Comment by MichaelCropper** [comment]
And doing a git diff on Master vs gh-pages branches shows it's just one single character difference. HTTP --> HTTP**S**
